packages/composer-popover/reducer.js

```diff
@@ -0,0 +1,61 @@
+import keyWrapper from '@bufferapp/keywrapper';
+import { actionTypes as orgActionTypes } from '@bufferapp/publish-data-organizations';
+
+export const actionTypes = keyWrapper('COMPOSER_POPOVER', {
+  OPEN_COMPOSER: 0,
+  OPEN_COMPOSER_VIA_BUTTON: 0,
+  OPEN_COMPOSER_VIA_TEXTBOX: 0,
+  HIDE_COMPOSER: 0,
+});
+
+export const actions = {
+  handleCreatePostClick: () => ({
+    type: actionTypes.OPEN_COMPOSER_VIA_BUTTON,
```

Comment:
Can't believe I lost the entire comment 😅 
If you look at this: https://github.com/bufferapp/buffer-publish/blob/main/packages/queue/middleware.js#L233-L239, we are tracking the event of clicking. The name of the event is here: https://github.com/bufferapp/buffer-publish/blob/main/packages/constants/index.js#L21.  if we keep this as it is, we'll be messing the tracking results, as it's gonna track calendar buttons as being in the queue.

This might be what you are looking for:

```suggestion
    type: actionTypes.OPEN_COMPOSER,
```
I'd remove any reference to `OPEN_COMPOSER_VIA_BUTTON` and `OPEN_COMPOSER_VIA_TEXTBOX` from this file, and handle those in the further refactor